test/infrastructure/docker/internal/loadbalancer/doc.go

```diff
@@ -14,5 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package loadbalancer contains external loadbalancer related constants and configuration
+// Package loadbalancer contains external loadbalancer related constants and configuration.
+// This package was originally forked from the kind load balancer at https://github.com/kubernetes-sigs/kind/tree/v0.7.0/pkg/cluster/internal/loadbalancer
```

Comment:
Not sure if this is the correct level of attribution, open to different opinions here for sure.

Reply:
Probably it's fine given that we copy from kubernetes-sigs to kubernetes-sigs.

If you want to make sure I would ask Ben.

Reply:
Kubernetes has done this sort of thing before, including in KIND!

IMHO we have a nice shortcut here in that these projects are both under the same license and "owner" (the project / contributors) so it's fine to just copy. I recommend attribution comments like this both for historical context and to acknowledge that another project originally wrote them.

So yes, this is fine to me. For prior art see also kubeadm/kinder tool 🙂

Reply:
Sounds good, thx for chiming in!